physics/noahmpdrv.meta

```diff
@@ -633,6 +633,20 @@
   dimensions = ()
   type = logical
   intent = in
+[cpllnd]
+  standard_name = flag_for_land_coupling
```

Comment:
We've been moving to `do_*` rather than `flag_for*`, but since `flag_for_land_coupling` already exists and there are several others like it, it seems unfair to ask you to change it everywhere. This will have to be part of a separate standard name cleanup at some later point, I guess.

Reply:
@grantfirl That is fine. I could fix them at least for land component coupling related once. When I look at to the examples for it, I found `do_mynnsfclay` but it's standard name is start with `flag_for`. Do you have any convention for it or just changing the argument name is fine at this point. Anyway, let me know what you think?

Reply:
Ya, I was talking about standard names only. I think that the variable names in the code are fine and the CCPP doesn't care. But, like I said, I wouldn't worry about changing it now since there is a lot of cleanup in other places to do anyway. The comment was more for the future, if you're going to be adding more on/off logicals, I think that having the standard names start with `do_X` is what the CCPP physics management committee prefers.

Reply:
@grantfirl Okay. I am not doing those changes at this point. Thanks.